src/istgt_lu_disk.c

```diff
@@ -5668,17 +5658,13 @@ check_for_command_completion(spec_t *spec, rcommon_cmd_t *rcomm_cmd, ISTGT_LU_CM
 		} else {
```

Comment:
this 'else' is not correct as explained in previous PR.
Lets say, copies_sent is 10, success is 1, failure is 9. In this case, success response is sent, which is not correct.
We atleast need (MAX(n-m+1, m)) number of success responses to start building the read response.

Reply:
I haven't added read consistency related changes in this PR.
I was thinking of handling this changes in US1320 PR.